src/nupic/research/connections.py

```diff
@@ -19,8 +19,7 @@
 # http://numenta.org/licenses/
 # ----------------------------------------------------------------------
 
-from collections import defaultdict, namedtuple
-
+from collections import defaultdict
 
 
 class SynapseData(object):
```

Comment:
I think we want 3 blank lines above this, not 2. "Separate top level functions and classes by three blank lines."